index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/styles/styles.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="/styles/mobile.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+<script src="https://kit.fontawesome.com/8a4d4148a1.js" crossorigin="anonymous"></script>
+  <title>Yasmina Mayeh</title>
+</head>
+<body>
+  <div class="page-container">
+    
+
+    <!-- HEADER -->
+    <header>
+      <nav>
+
+        <div class="mobile-burgerbar-container" onclick="mobileMenu()">
+          <span></span>
+        </div>
+
+        <p><span class="Yasmin-logo">YASMIN</span><span class="Mayeh-logo"> Mayeh</span></p>
+        <ul>
+          <li><a href="">HOME</a></li>
+          <li><a href="">ABOUT</a></li>
+          <li><a href="">SETTING</a></li>
+          <li><a href="">STORE</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!-- HEADER -->
+
+    <!-- THIS IS A MOBILE MENU -->
+    <div id="mobile-menu-container" >
+      <ol>
+        <li>About Yas</li>
+        <li>Photos</li>
+        <li>Books</li>
+      </ol>
+    </div>
+    <!-- THIS IS A MOBILE MENU -->
+
+    
+    <main>
+      <!-- THIS IS A POPUP MENU -->
+      <div class="pop-up-menu" id="popup-menu">
+        <menu>
+          <div class="x-container" onclick="closemenu()">
+            <div class="x-lines"></div>
+          </div>
+        </menu>
+        <article id="books">
+          <h1>My Favourite books</h1>
+          <p> <br> I like reading books non fiction mostly, but i also enjoy a bit of fiction here and there. My current read at the moment is ‘The little by Hanya Yanagihara’
+            Here are a couple books  on my to read list that i can share: The unexpected and The Bee Keeper of Allepo 
+          </p>
+        </article>
+
+        <article id="photos">
+          <div class="top-bottom">
+          <h1>Why i love photography</h1>
+          <p><blockquote>“I love how you can capture moments and reminisce about them even though it just happened.”</blockquote></p>
+          </div>
+          <div class="top-bottom">
+            <div class="popup-grid-container-photos">
+              <div class="box1"></div>
+              <div class="box2"></div>
+              <div class="box3"></div>
+
+            </div>
+          </div>
+        </article>
+
+        <article id="aboutyas">
```

Comment:
Kebab casing has been used with majority of class/id names. I would stick with this throughout to keep consistency and readability. 
```suggestion
        <article id="about-yas">
```